src/data/loadToDb.ts

```diff
@@ -0,0 +1,44 @@
+//import AWS from 'aws-sdk';
+import { productsDB } from './productsForLoad';
+import { stockDB } from './stockForLoad';
+import { IProduct, IStock, StatusCode } from '../types/types';
+import { generateHttpResponse } from '../utils/lambda';
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient, PutCommand } from "@aws-sdk/lib-dynamodb";
+
+const dynClient = DynamoDBDocumentClient.from(
+    new DynamoDBClient({ region: "eu-west-1" })
+  );
+const loadProductsToDB = async (products: IProduct[]) => {
+
+    if (process.env.PRODUCTS_TABLE === undefined) {
+
+        return generateHttpResponse(StatusCode.SERVER_ERROR, 'Something is wrong');
+    };
+    for (const product of products) {
+        const params = {
+            TableName: process.env.PRODUCTS_TABLE,
+            Item: product,
+        };
+        await dynClient.send(new PutCommand(params));
+
+    }
+};
+
+const loadStockToDB = async (stock: IStock[]) => {
+
+    if (process.env.STOCKS_TABLE === undefined) {
+
+        return generateHttpResponse(StatusCode.SERVER_ERROR, 'Something is wrong');
```

Comment:
It's one time script, not lambda function - we should not return HTTP response. This script is called manually from cli